src/components/Table.vue

```diff
@@ -415,10 +421,76 @@ export default {
 </style>
 
 <docs>
-### Basic Table
+### Tabela Básica
 
-Para alinhar o conteúdo das células da coluna em uma direção,
-adicione [center|right] como uma propriedade na coluna.
+A forma mais simples de utilizar o ViTable é apenas definir as colunas e dados que devem ser exibidos:
+
+> Dica: Para alinhar o conteúdo das células da coluna em uma direção,
+> adicione [center|right] como uma propriedade na coluna.
+
+```vue
+<template>
+  <vi-table :columns="columns" :items="data" />
+</template>
+
+<script>
+  export default {
+    data() {
+      return {
+        columns: [
+          { id: 'id', label: 'ID' },
+          { id: 'name', label: 'Name' },
+          { id: 'email', label: 'E-mail' },
+        ],
+        data: [
+          { id: 1, name: 'Amanda', email: 'amanda@vitta.com.br' },
```

Comment:
Recomendo usar um domínio fake, por causa de crawlers de email pra spam, mas não impede de aprovar o PR e mesclar na master.

Reply:
you are absolutely right! I'll fix it

Reply:
Done!